nature.md

```diff
@@ -751,7 +762,7 @@ Where user input errors are automatically detected by our own software, those er
 <br><br>
 Checkout process pages rely solely on built-in browser error checking, and do not identify errors in text. Some users with and without Assistive Technology may be unable to access these error states.
 <br><br>
-Third party scripts including the Disqus commenting widget, or other widgets that accept user input, may automatically detect errors but fail to present the errors to users in text. They may rely on visual characteristics alone to indicate user input errors, or place error text in locations that are difficult for screen reader users to access. When we become aware of deficiencies, we contact third party suppliers to request repairs.
+Third-party scripts that accept user input may automatically detect errors but fail to present the errors to users in text. They may rely on visual characteristics alone to indicate user input errors, or place error text in locations that are difficult for screen reader users to access. When we become aware of deficiencies, we contact third-party suppliers to request repairs.
```

Comment:
If the Freshdesk widget is indeed gone from Nature pages, you can remove this entirely - IIRC that was the only other example of this